scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -126,6 +126,15 @@ class PublishVisibility(Enum):
     UNLISTED = 'unlisted'
 
 
+class SystemRegisty:

```

Comment:
why do we need this and expose the subset of the registries here?

Reply:
I wanted to check a few things. there have been errors around model version. And if they get checked at workflow that would be great instead of mistakenly creating a drop and then creating a release and then understanding _"whoooa I did not update the model version."_

Anyway, there was issue with credential here. I would probably add that as separate PR and a gate that validates version for components and models and potentially env which are not auto versioned.

Reply:
@vizhur these are all public registries. Do you see any issue with specifying it here and cross-referencing details with what asset details in any of these registries?

Reply:
I've removed registry names as not needed in this PR. Will start discussion on teams for version check